include/openssl/rsa.h

```diff
@@ -215,6 +215,88 @@ OPENSSL_EXPORT int RSA_set0_factors(RSA *rsa, BIGNUM *p, BIGNUM *q);
 OPENSSL_EXPORT int RSA_set0_crt_params(RSA *rsa, BIGNUM *dmp1, BIGNUM *dmq1,
                                        BIGNUM *iqmp);
 
+// RSA_METHOD functions
+
+// RSA_get_default_method returns a zero initialized |RSA_METHOD| object. The
+// wrapper functions will select the appropriate |rsa_default_*| implementation.
+OPENSSL_EXPORT const RSA_METHOD *RSA_get_default_method(void);
+
+// RSA_meth_new returns a zero-initialized |RSA_METHOD| object. It sets
+// |flags| on the object. The |name| parameter is currently ignored and
+// part of the function signature for OpenSSL compatibility.
+OPENSSL_EXPORT RSA_METHOD *RSA_meth_new(const char *name, int flags);
```

Comment:
Yea we don't really need this functionality around flags yet so I am not implementing the associated functions either. Currently, only |RSA_FLAG_OPAQUE| is supported since it was offered before,  but there isn't necessarily a way to use it unless it's created with that flag in RSA_meth_new. Since this a previously offered flag, the necessary caveats/alternate codepaths already exist to accomodate this flag. we can always add this in later should we need it.